addons/base_vat/models/res_partner.py

```diff
@@ -39,7 +39,7 @@
     'de': 'DE123456788',
     'dk': 'DK12345674',
     'do': 'DO1-01-85004-3 or 101850043',
-    'ec': 'EC1792060346-001',
+    'ec': 'EC1792060346001 or EC1792060346',
```

Comment:
You don't want to remove the EC prefix?

Reply:
@EC is not part of the identifcatio number so better to remove it, and we should actually to the same with the other latam locs.

Reply:
@ren-odoo @jco-odoo the EC seems fine to me, because if the company is in Europe it seems to be common practice to use the EC prefix (at least it is what the source code suggests me), and if the company is in Ecuador then this warning message is never shown (we have a more detailed error that replaces this one in l10n_ec module)